src/pages/NotFound/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { Trans } from '@lingui/macro'
+import { Trace } from '@uniswap/analytics'
+import { SmallButtonPrimary } from 'components/Button'
+import { useIsDarkMode } from 'state/user/hooks'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import darkImage from '../../assets/images/404-page-dark.png'
+import lightImage from '../../assets/images/404-page-light.png'
+
+const Image = styled.img`
+  margin: 20px;
+  max-width: 510px;
+  width: 100%;
+  padding: 0 75px;
+`
+
+const Center = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media screen and (max-width: ${({ theme }) => theme.breakpoint.md}px) {
+    flex: 1;
+    justify-content: center;
+  }
+`
+
+const PageWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex: 1;
```

Comment:
Thanks to recent work, finally all it takes to render a full page layout is a simple `flex: 1` yay!